api/admin/controller.py

```diff
@@ -1504,11 +1506,36 @@ def custom_lists(self):
         if flask.request.method == "POST":
             id = flask.request.form.get("id")
             name = flask.request.form.get("name")
-            entries = flask.request.form.get("entries")
-            collections = flask.request.form.get("collections")
-            return self._create_or_update_list(library, name, entries, collections, id)
+            entries = self._getJSONFromRequest(flask.request.form.get("entries"))
+            collections = self._getJSONFromRequest(flask.request.form.get("collections"))
+            deletedEntries = self._getJSONFromRequest(flask.request.form.get("deletedEntries"))
```

Comment:
I think this is always creating a new list so `deletedEntries` wouldn't apply, but if I'm missing something and it is needed, `deleteAll` should also be handled here.

Reply:
No, that makes sense. If a new list is created then you'll only _really_ add new entries and the deleted ones we don't care about because those are just visually removed.